src/main/resources/static/ptmexchange/index.html

```
@@ -37,7 +37,7 @@ <h1>Datasets</h1>
         <td><i>Mus musculus</i></td><td>Mouse</td><td>Phospho</td><td>N/A</td><td><a href="https://peptideatlas.org/builds/mouse/phospho/">Mouse Phospho 2024&#8209;02</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000000589">Mus musculus (Mouse) proteome</a></td><td></td><td><a href="./Files/Mouse_GSB_phospo_all_prots_0125.csv.gz">  Results</a></td><td><a href="./Files/Mm_UP000000589-GCF_000001635.27-ENS105_cRAP_decoy.fasta.gz"> Fasta file</a></td>
       </tr>
       <tr class="pc-hov">
-        <td><i>Saccharomyces cerevisiae</i></td><td>Yeast</td><td>Phospho</td><td><a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD071918">PXD071918</a></td><td><a href="https://peptideatlas.org/builds/yeast/phospho/">Yeast Phospho 2024&#8209;08</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000002311">Saccharomyces cerevisiae (strain ATCC 204508 / S288c) (Baker's yeast) proteome</a></td><td></td><td><a href="./Files/Yeast_GSB_phospho_all_prots_0125.csv.gz">  Results</a></td><td> <a href="./Files/2023-11-27-decoys-contam-uniprotkb_proteome_UP000002311_2023_11_27.fasta.gz"> Fasta file</a></td>
+        <td><i>Saccharomyces cerevisiae</i></td><td>Yeast</td><td>Phospho</td><td><a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD071918">PXD071918</a></td><td><a href="https://peptideatlas.org/builds/yeast/phospho/">Yeast Phospho 2024&#8209;08</td><td>Visible on <a href="https://www.uniprot.org/proteomes/UP000002311">Saccharomyces cerevisiae (strain ATCC 204508 / S288c) (Baker's yeast) proteome</a></td><td><a href="https://pubs.acs.org/doi/10.1021/acs.jproteome.5c01263">Boswell et al., 2026</a></td><td><a href="./Files/Yeast_GSB_phospho_all_prots_0125.csv.gz">  Results</a></td><td> <a href="./Files/2023-11-27-decoys-contam-uniprotkb_proteome_UP000002311_2023_11_27.fasta.gz"> Fasta file</a></td>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </a> ], start tag match failed [ <a href="https://peptideatlas.org/builds/yeast/phospho/"> ] on line 40.

(tag-pair)